### DISCUSSION TITLE
It/sec is very slow for me, the speed is well below the example on the main page

### DISCUSSION THREAD

**Discussion by iterhating** [lllyasviel/Fooocus#168] [General]
Thanks for making this! I'm interested in testing it out more. Slight problem though, my computer is taking forever to make images.

Running windows 10, 32 GB RAM, 6 GB VRAM (GTX1660), I'm only getting ~6 sec/it after a ~4 minute text decoding time, no advanced settings.

Compare to the 1.35 seconds per iteration from a 16GB System RAM and 6GB VRAM (Nvidia 3060 laptop) on the main page.

I usually run Stable Diffusion in Auto1111 and just downloaded this today. With Auto1111 my computer usually runs ~2 sec/it. I'm not sure what I'm doing wrong or have misconfigured (or if it's just the sdxl's larger image size), I used the download/bat install as per the instructions.

Any help would be appreciated, current performance is probably too slow for me to continue trying with.

**Comment by lllyasviel** [comment]
yes 6 sec/it is slow. Have you tried the XL in a1111 on that machine? Is that "Auto1111 my computer usually runs ~2 sec/it" for XL?

**Comment by iterhating** [comment]
I don't know, unfortunately. Auto1111 won't run XL, I just get error messages. I checked out of the SD game for a few months and am just getting back into the swing of things, so I've probably just got to learn more about the new models/configurations. I just ran a 1.5 based model at 512x512 and it got 1.3 it/sec. So, it's possible the speed is just due to the size, I can't run at the sizes Fooocus lists inside Auto1111 without memory error or using hi-res fix which I don't think exactly scales.

Anyway, thanks for making it. Hopefully I and or it'll improve in time.

**Comment by realenuf** [comment]
I'm having the same issues on Ubuntu 22.04, with a 3060 12g vram. The images that generate eventually are impressive, and I can't get XL working on A111, SDnext, or InvokeAI. Fooocus takes a very long time, and often finally ends by killing the terminal window.
